src/Integration/Binding/RuleSetFileSystem.cs

```diff
@@ -0,0 +1,24 @@
+//-----------------------------------------------------------------------
+// <copyright file="RuleSetFileSystem.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
+
+namespace SonarLint.VisualStudio.Integration.Binding
+{
+    internal sealed class RuleSetFileSystem : IRuleSetFileSystem
```

Comment:
there are no tests for this file since it's a thin wrapper over the ruleset and shared between the solution and project binding operations.